testsuite/tests/contrib/lkt_semantic/array_type/test.lkt

```diff
@@ -0,0 +1,45 @@
+# TODO: support @invalid
+
+val a: Array[Int] = []
+val b: Array[Int] = [1]
+val c: Array[Int] = [1, 2, 3]
+
+# @invalid val a = []
```

Comment:
It’s unclear when reading this why this is commented: don’t we already have a diagnostic for this case? (same for the next)

Reply:
Ah indeed. Yes, even though `@invalid` is ignored, it looks like the test driver still has the behavior we want (print the error when there is one), so it’s ok.